### DISCUSSION TITLE
[Q&A] Latex 显示异常

### DISCUSSION THREAD

**Discussion by laike** [lobehub/lobehub#502] [General Question] [answered]
### 🥰 需求描述 | Feature Description

希望增加数学公式的解析 比如极限的符号无法正常显示 比如 Latex 

### 🧐 解决方案 | Proposed Solution

加入 latex markdown 插件

![LobeChat_自定义助手_2023-11-20](https://github.com/lobehub/lobe-chat/assets/3166610/1d888664-0a9f-41ae-b209-503722d6b799)

**Comment by canisminor1990** [comment, marked as answer]
Latex需要标记才能识别

![Screenshot_2023-11-20-20-15-14-891_com.android.chrome.jpg](https://github.com/lobehub/lobe-chat/assets/17870709/7990d419-0752-4494-85d1-8aff2f56e2d0)



```md
以下是一段Markdown格式的LaTeX数学公式：

我是一个行内公式：$E=mc^2$

我是一个独立公式：
$$
\sum_{i=1}^{n} x_i = x_1 + x_2 + \ldots + x_n
$$

我是一个带有分式的公式：
$$
\frac{{n!}}{{k!(n-k)!}} = \binom{n}{k}
$$

我是一个带有上下标的公式：
$$
x^{2} + y^{2} = r^{2}
$$

我是一个带有积分符号的公式：
$$
\int_{a}^{b} f(x) \, dx
$$
```

**Comment by canisminor1990** [comment]
你可在 prompt 中约定格式
